frontend/src/components/ConcResultTable.tsx

```diff
@@ -1,20 +1,10 @@
 import { Table } from "@equinor/eds-core-react";
 
-type InitFinalDiff = {
-    initFinalDiff: {
-        initial: {
-            [key: string]: number;
-        };
-        final: {
-            [key: string]: number;
-        };
-        change: {
-            [key: string]: number;
-        };
-    };
-};
```

Comment:
anyone have any thoughts of this part? I think I made the change to not use `initFinalDiff`, as the change is always implicit, and it adds another layer. At the same time we need to recalculate the diff relatively often..

Personally I prefer to keep the change (using `initialConcentrations` etc), but if everyone else is opposed I won't have (too) strong opinions.. :D